CHANGELOG.md

```diff
@@ -68,6 +68,7 @@
 - Add syntax mapping for kubernetes config files #3049 (@cyqsimon)
 - Adds support for pipe delimiter for CSV #3115 (@pratik-m)
 - Add syntax mapping for `/etc/pacman.conf` #2961 (@cyqsimon)
+- Associate `uv.lock` with `TOML` syntax, see #3130 (@fepegar)
```

Comment:
Regarding your comment on [#3130](https://github.com/sharkdp/bat/issues/3130#issuecomment-2495048651):
Changelog entries are supposed to contain the PR number or, optionally, both PR and issue numbers ([CONTRIBUTING.md](https://github.com/sharkdp/bat/blob/master/CONTRIBUTING.md#changelog-entry-format) provides further information). So something like:
```suggestion
- Associate `uv.lock` with `TOML` syntax, see #3132 (@fepegar)
```
or
```suggestion
- Associate `uv.lock` with `TOML` syntax, see issue #3130 and PR #3132 (@fepegar)
```
should work.

Reply:
Ah! Apologies for missing that and thanks for your patience. Let me change that.